conda-recipe/meta.yaml

```diff
@@ -13,6 +13,8 @@ requirements:
       - cmake
       - dpctl >=0.5.0a0
       - mkl-devel >=2021.1.1
+      - intel-openmp
+      - tbb
```

Comment:
@Alexander-Makaryev  please confirm this but I think the project doesn't depend on TBB and OpenMP

Reply:
Do we really need this pkgs? Properly check is needed.

Reply:
This is an attempt to solve internal build issue, dependencies are from MKL not DPNP. Issue is not a critical/stopper/etc. So I think we should remove them from here and find another way.

Reply:
Ok, I will remove this packages